internal/serverinstall/nomad.go

```diff
@@ -67,6 +67,7 @@ type nomadConfig struct {
 	csiTopologies        map[string]string `hcl:"nomad_csi_topologies,optional"`
 	csiSecrets           map[string]string `hcl:"nomad_csi_secrets,optional"`
 	csiParams            map[string]string `hcl:"csi_parameters,optional"`
+	csiVolumeName        string            `hcl:"nomad_csi_volume_name,optional"`
```

Comment:
same comment here as in the runner install file.

Reply:
Resolved in https://github.com/hashicorp/waypoint/pull/3546/commits/82a595eb404e8bd86ba9d7b6276ba4e894ac9137